src/xunit.performance.api/Profilers/Etw/PmcRollover.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Xunit.Performance.Api.Profilers.Etw
+{
+    /// <summary>
+    /// Defines a performance monitor sample.
+    /// </summary>
+    internal sealed class PmcRollover
```

Comment:
Nit: I'd recommend calling this a PmcSample instead of a Rollover.  It took me a bit of time to realize that (I think) you mean that this object represents a PMC event that is fired once the measured metric threshold is reached.

Reply:
Yes, that is exactly what it is.

---
In reply to: [146052342](https://github.com/Microsoft/xunit-performance/pull/242#discussion_r146052342) [](ancestors = 146052342)